controllers/server.go

```diff
@@ -18,16 +19,24 @@ func serverHandlers(r *mux.Router) {
 	// r.HandleFunc("/api/server/addnetwork/{network}", securityCheckServer(true, http.HandlerFunc(addNetwork))).Methods(http.MethodPost)
 	r.HandleFunc(
 		"/api/server/health",
-		http.HandlerFunc(func(resp http.ResponseWriter, req *http.Request) {
+		func(resp http.ResponseWriter, req *http.Request) {
 			resp.WriteHeader(http.StatusOK)
 			resp.Write([]byte("Server is up and running!!"))
-		}),
-	)
+		},
+	).Methods(http.MethodGet)
+	r.HandleFunc(
+		"/api/server/health",
```

Comment:
change api route

Reply:
any suggestions? i found acceptable to have GET server/health and DELETE server/health to check if its on or turn it off haha

Reply:
https://github.com/gravitl/netmaker/pull/2672/commits/b1ece5f72e3aae7bb4ecb5a9885bd8b87785150d